site/content/concepts/command-and-control.md

```diff
@@ -66,3 +66,24 @@ The response of the command will be sent by the device to
 If the device is authenticated, the `tenant` and `device-id` are left empty (resulting in 3 subsequent `/`s).
 
 ![Command & Control over MQTT Adapter](../command_control_concept_mqtt.png) 
+
+## Command & Control over AMQP Adapter
+
+When an authenticated device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link at the source address:
+
+* `control`
+
+while unauthenticated devices have to fully specify their `tenant` and `device-id` as source address. 
+
+Once the link is opened, the device can receive command messages from the adapter. The AMQP adapter sets the
+`reply-to` and `subject` properties of the message as follows:
+
+* `reply-to`: `control///req/<req-id>`(authenticated device) or `control/tenant/device-id/<req-id>` (unauthenticated device). The `<req-id>` can be used by the device when sending the command response to the adapter.
```

Comment:
I think this is wrong: I would set the `req-id` as property, which is then set again by the device when responding and  using it's `sender` link that has opened the `control` endpoint.
And the AMQP protocol adapter uses it to send the message to the application by using the endpoint `control/tenant/device-id/${reply-id}`. Remark: the reply-id is constructed from the req-id.
WDYT?

Reply:
yes, i see! however, when publishing messages (events, telemetry, command responses) to the adapter, devices open an anonymous sender link (with no target address). The address is set in the AMQP message address

Reply:
> Remark: the reply-id is constructed from the req-id.
WDYT?

yes, i think this is the default for all protocol adapters i.e command responses are sent to `control/tenant/device-id/${reply-id}`?

Reply:
Ok, I am probably not deep enough into the AMQP protocol adapter yet, but I still do not understand why the complete `reply-to` property is important for the device?

IMHO only the `req-id` and the `subject` need to be set, right?
No matter if the sender link is anonymous or set with an address...

Reply:
> I still do not understand why the complete reply-to property is important for the device?
IMHO only the req-id and the subject need to be set, right?

yes, that's correct. it makes a lot of sense to not set the `reply-to` property and only set the `req-id` and `subject`..  I was thinking you meant from device -> adapter